openedx/core/djangoapps/appsembler/settings/settings/test_common.py

```diff
@@ -42,3 +42,5 @@ def plugin_settings(settings):
     settings.FEATURES['TAHOE_SITES_USE_ORGS_MODELS'] = False
 
     settings.TAHOE_NUTMEG_TEMP_SKIP_TEST = True
+
+    settings.DEFAULT_COURSE_MODE_SLUG = 'audit'
```

Comment:
We use honor code to enable certificates.

```suggestion
    settings.DEFAULT_COURSE_MODE_SLUG = 'honor'
```

Reply:
Done